### PR TITLE
New fields

### PR DESCRIPTION
**Pull request by Ni-Knight**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-10021

## Description
New indicator fields that are missing from XSOAR for STIX2.x

Number of subkeys for Registry Key
image

Malware sample references
image

### CODE REVIEW

File: Packs/CommonTypes/IndicatorFields/indicatorfield-numberofsubkeys.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "content": true,
+    "group": 2,
+    "hidden": false,
+    "openEnded": false,
+    "associatedTypes": [
+        "Registry Key"
+    ],
+    "associatedToAll": false,
+    "unmapped": false,
+    "unsearchable": false,
```
Comment (ssokolovich) [inline Packs/CommonTypes/IndicatorFields/indicatorfield-numberofsubkeys.json:26]: this should be searchable or not?

File: Packs/CommonTypes/IndicatorFields/indicatorfield-samples.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    "content": true,
+    "group": 2,
+    "hidden": false,
+    "openEnded": false,
+    "associatedTypes": [
+        "Malware"
+    ],
+    "associatedToAll": false,
+    "unmapped": false,
+    "unsearchable": false,
```
[the remark this answers is not in the document]
Comment (Ni-Knight) [reply Packs/CommonTypes/IndicatorFields/indicatorfield-samples.json:26]: Formatter default is set to False due to performance issues, that is a call by higher ups.

File: Packs/CommonTypes/IndicatorFields/indicatorfield-numberofsubkeys.json (hunk 1 above)
Comment (Ni-Knight) [reply Packs/CommonTypes/IndicatorFields/indicatorfield-numberofsubkeys.json:26]: Formatter default is set to False due to performance issues, that is a call by higher ups.

File: Packs/CommonTypes/IndicatorFields/indicatorfield-samples.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (Ni-Knight) [reply Packs/CommonTypes/IndicatorFields/indicatorfield-samples.json:26]: Yes

File: Packs/CommonTypes/IndicatorFields/indicatorfield-numberofsubkeys.json (hunk 1 above)
Comment (Ni-Knight) [reply Packs/CommonTypes/IndicatorFields/indicatorfield-numberofsubkeys.json]: no, fixed

**Review by ssokolovich — APPROVED** [review]

**Pull request merged**